rviz_aerial_plugins/src/rviz_aerial_plugins/displays/goal3D/goal3D_display.cpp

```diff
@@ -124,21 +124,21 @@ void Goal3DDisplay::onInitialize()
 
   auto timer_callback =
      [this]() -> void {
-       visualization_msgs::msg::InteractiveMarker int_marker;
-       server_->get("marker", int_marker);
+       visualization_msgs::msg::InteractiveMarker rviz_aerial_goal3D_marker;
+       server_->get("marker", rviz_aerial_goal3D_marker);
```

Comment:
Sorry when I mentioned changing the name I was talking about the name attribute of the"marker" not the int_marker variable name. The variable name is local to the instance and the more verbose is a little clearer. But the marker.name is broadcast over the whole system and as such is what's vulnerable to colliding. If someone else uses a marker named "marker" for a different plugin they would then potentially cross talk in weird ways.

Reply:
my bad. solved

Reply:
shoiuld I use `constexpr` or a `define` for the magic number?